apps/webapp/app/presenters/JobPresenter.server.ts

```diff
@@ -0,0 +1,26 @@
+import { User } from "~/models/user.server";
+import { PrismaClient, prisma } from "~/db.server";
+import { Job } from "~/models/job.server";
+
+export class JobsListPresenter {
+  #prismaClient: PrismaClient;
+
+  constructor(prismaClient: PrismaClient = prisma) {
+    this.#prismaClient = prismaClient;
+  }
+
+  public async call({
+    slug,
+    userId,
+  }: {
+    slug: string;
+    userId: string;
+  }): Promise<Job[]> {
+    return this.#prismaClient.job.findMany({
+      where: {
+        projectId: slug,
+        id: userId,
```

Comment:
This is still not right

> You are querying here for many jobs but you are trying to find the job by the user ID. The query here should return the same exact type that is being returned from the ProjectPresenter.call, scoped to just the jobs.

Reply:
Dang! I must have missed this. I'll fix now.

Reply:
Just to be clear again, I'll have to follow this pattern from the ProjectPresenter and use `findFirst`, yeah?

```ts
 public async call({
    userId,
    slug,
  }: Pick<Project, "slug"> & {
    userId: User["id"];
  }) {
 ```

Reply:
For now, I'm only returning jobs that are not internal. 

```ts
 return project.jobs.filter((job) => !job.internal);
```  
Is it okay this way?